This is necessary for find_library_file to work where a library has only
an implib but no static lib (e.g. sqlite3).

--- Python-3.2.3/Lib/distutils/cygwinccompiler.py.orig	2012-04-11 01:54:02.000000000 -0500
+++ Python-3.2.3/Lib/distutils/cygwinccompiler.py	2012-06-22 19:37:11.740576800 -0500
@@ -92,9 +92,7 @@ class CygwinCCompiler(UnixCCompiler):
     compiler_type = 'cygwin'
     obj_extension = ".o"
     static_lib_extension = ".a"
-    shared_lib_extension = ".dll"
-    static_lib_format = "lib%s%s"
-    shared_lib_format = "%s%s"
+    shared_lib_extension = ".dll.a"
     exe_extension = ".exe"
 
     def __init__(self, verbose=0, dry_run=0, force=0):
--- Python-3.2.5/Lib/distutils/unixccompiler.py.orig	2013-07-30 16:39:21.092550800 -0500
+++ Python-3.2.5/Lib/distutils/unixccompiler.py	2013-07-30 20:01:52.168552600 -0500
@@ -81,6 +81,7 @@ class UnixCCompiler(CCompiler):
     xcode_stub_lib_format = dylib_lib_format
     if sys.platform == "cygwin":
         exe_extension = ".exe"
+        dylib_lib_extension = ".dll.a"
 
     def preprocess(self, source, output_file=None, macros=None,
                    include_dirs=None, extra_preargs=None, extra_postargs=None):
